README.md

```diff
@@ -1,2 +1,26 @@
-# Mobile-Version
-Portfolio: mobile version
+# Portfolio
+Set up and mobile version skeleton
+
+## Built With
```

Comment:
- [OPTIONAL] You did a really good job on adding Readme file but I suggest you to add a screenshot of your project on the Readme file to make it perfect!